.gitignore

```diff
@@ -117,3 +117,4 @@ structure.sql
 
 # Local development direnv file
 /.envrc
+docs/getting-started/my-account/EmailNotificationsv12.png
```

Comment:
Here a picture has accidentally found its way into the git ignore file. Please remove this line, so there are no changes to this file.